hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -153,8 +153,14 @@ private void processEvent(int cfIndex, byte[] keyBytes, byte[]
           }
         }
       } else if (action.equals(DELETE)) {
-        if (oldValue != null && !omUpdateEventValidator.isValidEvent(tableName,
-            oldValue, key, action)) {
+        if (null == oldValue) {
+          String keyStr = (key instanceof String) ? key.toString() : "";
+          LOG.warn("Value of Key: {} in table: {} should not be null " +
```

Comment:
In case that the key is not a string, we are logging `"Value of Key: in table: 'tableName' should not be null".` Could you please improve it a bit more so the person encountering the log message understand what happened here?

Reply:
@Galsza Thanks for re-review. I have added bit more information in case of key as non-String type. Pls check and review once.